### PR TITLE
little bit more code coverage

### PR DESCRIPTION
**Pull request by Uzlopak**
a little bit more code coverage

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: lib/types/multipart.js [hunk 1]
```diff
@@ -280,15 +278,16 @@ Multipart.prototype.end = function () {
       self._boy._done = true
       self._boy.emit('finish')
     })
-  } else if (this.parser.writable) { this.parser.end() }
```
Comment (kibertoad) [inline lib/types/multipart.js:283]: why is this condition not needed?

Comment (Uzlopak) [reply lib/types/multipart.js]: I thought writable means it checks if it is a writable stream, but after checking the documentation, it seems to mean that it was not destroyed, reverted that

File: lib/types/multipart.js [hunk 2]
```diff
@@ -288,7 +287,6 @@ function skipPart (part) {
 }
 
 function FileStream (opts) {
-  if (!(this instanceof FileStream)) { return new FileStream(opts) }
```
Comment (kibertoad) [inline lib/types/multipart.js:291]: isn't this helpful?

Comment (Uzlopak) [reply lib/types/multipart.js:291]: not really. we use this Stream internally and we call it with 'new' operator in line 172. So this line can never be active, and it is not possible to unit test it except we export the function, which makes no sense, as it is not meant to be used externally.

**Review by kibertoad — APPROVED** [review]

**Pull request merged**